client/src/hooks/useTripQueryVariables.ts

```diff
@@ -1,10 +1,11 @@
 import { useEffect, useState } from 'react';
-import { TripQueryVariables } from '../gql/graphql.ts';
+import { Locale, TripQueryVariables } from '../gql/graphql.ts';
 
 const DEFAULT_VARIABLES: TripQueryVariables = {
   from: {},
   to: {},
   dateTime: new Date().toISOString(),
+  locale: Locale.Us,
```

Comment:
I agree with this PR except this change (which doesn't belong in this PR anyway). Split it out so we can consider it separately?